plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -416,6 +424,7 @@ void producer_plugin::set_program_options(
 
    producer_options.add_options()
          ("enable-stale-production,e", boost::program_options::bool_switch()->notifier([this](bool e){my->_production_enabled = e;}), "Enable block production, even if the chain is stale.")
+         ("enable-stale-speculation", boost::program_options::bool_switch()->notifier([this](bool e){my->_enable_stale_speculation = e;}), "Enable speculative execution and relay, even if the chain is stale.")
```

Comment:
I don't see any reason to include this option.

Reply:
I'm leaning towards agreeing.  It is only present to maintain the original default functionality however, there seems to be almost zero value in that mode of operation.